staging/src/k8s.io/apiextensions-apiserver/test/integration/basic_test.go

```diff
@@ -93,7 +93,8 @@ func testSimpleCRUD(t *testing.T, ns string, noxuDefinition *apiextensionsv1beta
 	for _, v := range noxuDefinition.Spec.Versions {
 		noxuResourceClients[v.Name] = newNamespacedCustomResourceVersionedClient(ns, dynamicClient, noxuDefinition, v.Name)
 
-		noxuWatch, err := noxuResourceClients[v.Name].Watch(metav1.ListOptions{})
+		noxuWatch := noxuResourceClients[v.Name].Watch(metav1.ListOptions{})
```

Comment:
Why those are related to etcd upgrade?

Reply:
No. The watch function does not return err.